staging/src/k8s.io/kubectl/pkg/cmd/version/version.go

```diff
@@ -101,7 +105,10 @@ func (o *Options) Complete(f cmdutil.Factory, cmd *cobra.Command) error {
 }
 
 // Validate validates the provided options
-func (o *Options) Validate() error {
+func (o *Options) Validate(args []string) error {
```

Comment:
Function signature might be changed according to it https://github.com/kubernetes/kubernetes/blob/9b09612d1b0a6ec441833f928cbb205d936da59c/staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go#L335

Reply:
Fixed

Reply:
This is wrong place, this is causing tight coupling wit args, `Validate` is meant to ONLY to verify arguments of `Options` struct nothing outside. If you want to introduce such validation this should be done in `Complete` similarly to https://github.com/kubernetes/kubectl/blob/4da03973dd2fcd4645f20ac669d8a73cb017ff39/pkg/cmd/apiresources/apiversions.go#L72-L74

Reply:
Thanks for the comments. If no object, I will submit another PR to fix it.
cc @eddiezane